dags/dag_content/tower_metrics_content.py

```diff
@@ -1,21 +1,20 @@
-import os
 import json
+import os
 import time
 from itertools import chain
 from typing import Any
 
-from airflow.decorators import task
-from airflow.models import Variable
-import boto3
 import synapseclient
+from airflow.decorators import task
+from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
+from airflow.providers.amazon.aws.hooks.rds import RdsHook
+from airflow.providers.amazon.aws.hooks.secrets_manager import SecretsManagerHook
+from dag_content.utils import create_synapse_session
 
 # VARIABLES
 
-# AWS creds
-AWS_CREDS = {
-    "AWS_ACCESS_KEY_ID": Variable.get("TOWER_DB_ACCESS_KEY"),
-    "AWS_SECRET_ACCESS_KEY": Variable.get("TOWER_DB_SECRET_ACCESS_KEY"),
-}
+# AWS conn
+AWS_CONN_ID = "TOWER_DB_CONNECTION"
```

Comment:
**Nit (_i.e._ [optional](https://google.github.io/eng-practices/review/reviewer/standard.html#:~:text=Reviewers%20should%20always%20feel%20free,they%20could%20choose%20to%20ignore.)):** If this works, then we don't need to change anything, but I think the connection ID is a good candidate to be a DAG parameter ([docs](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/params.html)). This way, it can be overridden on launch.

Reply:
The way this is currently structured, with the DAG tasks in a separate script, the `AWS_CONN_ID` and the hooks never actually touch the DAG script itself so parameterizing isn't really possible I don't think. We would either need to import the hooks to the DAG or provide the `aws_conn_id` as a parameter to each task